setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "0.4.1"
+VERSION = "0.4.2"
```

Comment:
This version should be bumped to the next major version, especially since a new resource type is being added besides new APIs.

Reply:
well, major would be 1.0.0... But I agree that this seems big enough for a 0.5.0.